### PR TITLE
[Python3.9] Add support for Python 3.9 on RHEL8 and RedHat8-ARM64

### PR DESCRIPTION
**Pull request by agl29**
## What changes were proposed in this pull request?

- RHEL8 Py3.8 to Py3.9
- RHEL9-ARM64 Py3.9

## How was this patch tested?

- Built Hue on Cloudera docker images
- CDH Canary builds

### CODE REVIEW

File: tools/cloudera/build_hue_common.sh [hunk 1]
```diff
@@ -296,8 +296,39 @@ function redhat8_install() {
...
+      ncurses-devel \
+      nmap-ncat \
+      xmlsec1 \
+      xmlsec1-openssl \
+      libss \
+      ncurses-c++-libs'
+    # MySQLdb install
+    sudo -- sh -c 'yum install -y python3-mysqlclient'
+    # NODEJS 20 install
+    sudo -- sh -c 'curl -fsSL https://rpm.nodesource.com/setup_lts.x | sudo bash - && \
```
Comment (Harshg999) [inline tools/cloudera/build_hue_common.sh:327]: Will this always give node v20? Even after some new future supported version is available?

**Review by Harshg999 — APPROVED** [review]
LGTM!

File: tools/cloudera/build_hue_cloudera.sh [hunk 2]
```diff
@@ -77,6 +77,8 @@ function install_prerequisite() {
     redhat9_install
   elif [[ $1 == "ubuntu22" ]]; then
     ubuntu22_install
+  elif [[ $1 == "redhat8-arm64" ]]; then
+    redhat8_arm64_install
```
[the remark this answers is not in the document]
Comment (agl29) [reply tools/cloudera/build_hue_cloudera.sh:81]: Currently, both functions are the same, but in the future, there might be different dependencies or the possibility that we will support different Python versions on each OS. For multi-Python support, we may also need to make adjustments specifically for Red Hat 8. With that in mind, I believe it's better to maintain separate functions for each OS.

File: tools/cloudera/build_hue_common.sh (hunk 1 above)
Comment (amitsrivastava) [reply tools/cloudera/build_hue_common.sh:327]: As @Harshg999 pointed out, this is a fragile change. If we do not want to inadvertently upgrade the node versions, then we need to lock it in. Let's get this one in as is.
@athithyaaselvam, can you take up the node version issue and clean things up in a separate commit if needed?

**Review by amitsrivastava — APPROVED** [review]

**Pull request merged**